community-plugins.json

```diff
@@ -12430,5 +12430,14 @@
         "author": "Minh Vương",
         "description": "Auto rename pasted images added to the vault via hash algorithm SHA-512",
         "repo": "hardingadonis/hash-pasted-image"
-    }
+    },
+    {
+	"id": "text-composer-for-obsidian",
+	"name": "Text Composer",
+	"author": "Your Name",
```

Comment:
Hi @filipaldi Looks like you filled this out incorrectly. Your name is meant to go here